modules/contentpool_remote_register/contentpool_remote_register.module

```diff
@@ -1,2 +1,25 @@
 <?php
 
+use Drupal\Core\Entity\EntityInterface;
+
+const CONTENTPOOL_CONTENT_ENTITIES = ['node', 'media', 'file', 'paragraph', 'taxonomy_term'];
+
+/**
+ * Implements hook_entity_insert().
+ *
+ * @param \Drupal\Core\Entity\EntityInterface $entity
+ */
+function contentpool_remote_register_entity_insert(EntityInterface $entity) {
+  if (in_array($entity->getEntityTypeId(), CONTENTPOOL_CONTENT_ENTITIES)) {
+    \Drupal::service('contentpool_remote_register.registration_pull_manager')->pullFromRemoteRegistrations();
```

Comment:
I think we need to move the pushing to the end of the page request, like search api does with indexing. -> I think it uses a shutdown function. Let's do that also.